SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1988,7 +1988,7 @@ void USpatialNetDriver::TickFlush(float DeltaTime)
 #endif // WITH_SERVER_CODE
 	}
 
-	if (SpatialGDKSettings->UseRPCRingBuffer() && Sender != nullptr)
+	if (Sender != nullptr)
```

Comment:
Can assume this to be true?

Reply:
No, the net driver can tick before the Spatial connection is established (which is when Sender is created)